### DISCUSSION TITLE
How many years have you been programming? Can you support TEA?

### DISCUSSION THREAD

**Discussion by vu8690** [pkgxdev/pkgx#773] [Help]
How many years have you been programming? Can you support TEA?
With the experience you have gained, let's exchange and help each other.

**Comment by Adehex** [comment]
It barely a year since I started programming

**Comment by Adehex** [comment]
I can support Tea

**Reply by gdmnkam** [reply]
I'm not a coder, but many years i worked in IT departments as support/manager and chief

**Reply by gdmnkam** [reply]
But of course o can support tea as A,bassador, CM. content creator, etc

**Comment by sambojix** [comment]
If you are interested and want to join, send an application containing your experience in this matter, then the team will provide an answer.
![Uploading Screenshot_2024-01-10-22-16-31-93_40deb401b9ffe8e1df2f1cc5ba480b12.jpg…]()

**Reply by sambojix** [reply]
In Discord, you can find important pages under #official-links

**Comment by Starkexrein** [comment]
Just started but hope to learn a lot this year :)
